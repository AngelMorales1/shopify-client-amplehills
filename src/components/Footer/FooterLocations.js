import React from 'react';
import PropTypes from 'prop-types';
import cx from 'classnames';

import { sortHours } from 'utils/sortHours';
import FooterRegions from 'constants/FooterRegions';
import locationModel from 'models/locationModel';

import styles from './Footer.scss';

const FooterLocations = ({ locations }) => {
  const locationsSortedByRegion = (locations = []) => {
    const regions = locations.reduce((accumulated, current) => {
      let region = current.region;
      accumulated[region] = accumulated[region]
        ? accumulated[region].concat([current])
        : [current];

      return accumulated;
    }, {});

    return FooterRegions.reduce(
      (accumulated, region) => ({
        ...accumulated,
        [region]: regions[region]
      }),
      {}
    );
  };

  const regions = locationsSortedByRegion(locations);

  return (
    <div
      className={cx(
        'flex flex-column m4',
        styles['Footer__Locations-container']
      )}
    >
      <h2 className="mb2 text-white block-headline">Locations</h2>
      <div
        className={cx(
          'flex flex-column flex-wrap',
          styles['Footer__Regions-container']
        )}
      >
        {Object.keys(regions).map(region => (
          <div
            key={region}
            className={cx(
              'flex flex-column',
              styles['Footer__Regions-content']
            )}
          >
            <h3 className="my2 text-white callout">{region}</h3>
            {regions[region].map(location => {
              let hours = sortHours(location.hours);

              return (
                <div
                  className={cx('mb3', styles['Footer__Regions-store'])}
                  key={location.id}
                >
                  <h4 className="mb1 text-white bold small nowrap">
                    {location.title}
                  </h4>
                  {hours.map((hour, i) => {
                    const hourKey = Object.keys(hour);
                    return (
                      <p
                        className="mb1 text-white small nowrap"
                        key={i}
                      >{`${hourKey}: ${hour[hourKey]}`}</p>
                    );
                  })}
                  {location.delivery ? (
                    <div className="bg-white text-madison-blue inline-block mt1 nowrap tag">
                      Order Delivery
                    </div>
                  ) : null}
                </div>
              );
            })}
          </div>
        ))}
      </div>
    </div>
  );
};

export default FooterLocations;

FooterLocations.propTypes = {
  locations: PropTypes.arrayOf(locationModel.propTypes)
};

FooterLocations.defaultProps = {
  locations: []
};
