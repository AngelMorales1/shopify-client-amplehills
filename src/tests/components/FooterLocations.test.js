import React from 'react';
import { shallow } from 'enzyme';

import FooterLocations from 'components/Footer/FooterLocations';

it('renders without data', () => {
  const component = shallow(<FooterLocations />);

  expect(component).toMatchSnapshot();
});

it('renders with data', () => {
  const component = shallow(
    <FooterLocations
      locations={[
        {
          id: 'foo',
          title: 'foo',
          image: 'foo',
          seasonalImage: 'foo',
          address1: 'foo',
          address2: 'foo',
          city: 'foo',
          region: 'foo',
          state: 'foo',
          zip: 'foo',
          coordinates: {
            lon: 0,
            lat: 0
          },
          phone: 'foo',
          seasonal: false,
          hours: {
            monday: 'foo',
            tuesday: 'foo',
            wednesday: 'foo',
            thursday: 'foo',
            friday: 'foo',
            saturday: 'foo',
            sunday: 'foo'
          },
          sortedHours: [{ foo: 'foo' }],
          delivery: false
        }
      ]}
    />
  );

  expect(component).toMatchSnapshot();
});
