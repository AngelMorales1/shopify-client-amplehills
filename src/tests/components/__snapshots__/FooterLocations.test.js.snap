// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FooterLocations
    locations={
        Array [
            Object {
              "address1": "foo",
              "address2": "foo",
              "city": "foo",
              "coordinates": Object {
                "lat": 0,
                "lon": 0,
              },
              "delivery": false,
              "hours": Object {
                "friday": "foo",
                "monday": "foo",
                "saturday": "foo",
                "sunday": "foo",
                "thursday": "foo",
                "tuesday": "foo",
                "wednesday": "foo",
              },
              "id": "foo",
              "image": "foo",
              "phone": "foo",
              "region": "foo",
              "seasonal": false,
              "seasonalImage": "foo",
              "sortedHours": Array [
                Object {
                  "foo": "foo",
                },
              ],
              "state": "foo",
              "title": "foo",
              "zip": "foo",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="mb2 text-white block-headline"
>
          Locations
</h2>,
        <div
          className="flex flex-column flex-wrap"
>
          <div
                    className="flex flex-column"
          >
                    <h3
                              className="my2 text-white callout"
                    >
                              Brooklyn
                    </h3>
          </div>
          <div
                    className="flex flex-column"
          >
                    <h3
                              className="my2 text-white callout"
                    >
                              Manhattan
                    </h3>
          </div>
          <div
                    className="flex flex-column"
          >
                    <h3
                              className="my2 text-white callout"
                    >
                              Queens
                    </h3>
          </div>
          <div
                    className="flex flex-column"
          >
                    <h3
                              className="my2 text-white callout"
                    >
                              New Jersey
                    </h3>
          </div>
          <div
                    className="flex flex-column"
          >
                    <h3
                              className="my2 text-white callout"
                    >
                              Florida
                    </h3>
          </div>
</div>,
      ],
      "className": "flex flex-column m4",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Locations",
          "className": "mb2 text-white block-headline",
        },
        "ref": null,
        "rendered": "Locations",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="flex flex-column"
>
              <h3
                            className="my2 text-white callout"
              >
                            Brooklyn
              </h3>
</div>,
            <div
              className="flex flex-column"
>
              <h3
                            className="my2 text-white callout"
              >
                            Manhattan
              </h3>
</div>,
            <div
              className="flex flex-column"
>
              <h3
                            className="my2 text-white callout"
              >
                            Queens
              </h3>
</div>,
            <div
              className="flex flex-column"
>
              <h3
                            className="my2 text-white callout"
              >
                            New Jersey
              </h3>
</div>,
            <div
              className="flex flex-column"
>
              <h3
                            className="my2 text-white callout"
              >
                            Florida
              </h3>
</div>,
          ],
          "className": "flex flex-column flex-wrap",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "Brooklyn",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="my2 text-white callout"
>
                  Brooklyn
</h3>,
                Array [],
              ],
              "className": "flex flex-column",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Brooklyn",
                  "className": "my2 text-white callout",
                },
                "ref": null,
                "rendered": "Brooklyn",
                "type": "h3",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "Manhattan",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="my2 text-white callout"
>
                  Manhattan
</h3>,
                Array [],
              ],
              "className": "flex flex-column",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Manhattan",
                  "className": "my2 text-white callout",
                },
                "ref": null,
                "rendered": "Manhattan",
                "type": "h3",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "Queens",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="my2 text-white callout"
>
                  Queens
</h3>,
                Array [],
              ],
              "className": "flex flex-column",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Queens",
                  "className": "my2 text-white callout",
                },
                "ref": null,
                "rendered": "Queens",
                "type": "h3",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "New Jersey",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="my2 text-white callout"
>
                  New Jersey
</h3>,
                Array [],
              ],
              "className": "flex flex-column",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "New Jersey",
                  "className": "my2 text-white callout",
                },
                "ref": null,
                "rendered": "New Jersey",
                "type": "h3",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "Florida",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="my2 text-white callout"
>
                  Florida
</h3>,
                Array [],
              ],
              "className": "flex flex-column",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Florida",
                  "className": "my2 text-white callout",
                },
                "ref": null,
                "rendered": "Florida",
                "type": "h3",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="mb2 text-white block-headline"
>
            Locations
</h2>,
          <div
            className="flex flex-column flex-wrap"
>
            <div
                        className="flex flex-column"
            >
                        <h3
                                    className="my2 text-white callout"
                        >
                                    Brooklyn
                        </h3>
            </div>
            <div
                        className="flex flex-column"
            >
                        <h3
                                    className="my2 text-white callout"
                        >
                                    Manhattan
                        </h3>
            </div>
            <div
                        className="flex flex-column"
            >
                        <h3
                                    className="my2 text-white callout"
                        >
                                    Queens
                        </h3>
            </div>
            <div
                        className="flex flex-column"
            >
                        <h3
                                    className="my2 text-white callout"
                        >
                                    New Jersey
                        </h3>
            </div>
            <div
                        className="flex flex-column"
            >
                        <h3
                                    className="my2 text-white callout"
                        >
                                    Florida
                        </h3>
            </div>
</div>,
        ],
        "className": "flex flex-column m4",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Locations",
            "className": "mb2 text-white block-headline",
          },
          "ref": null,
          "rendered": "Locations",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="flex flex-column"
>
                <h3
                                className="my2 text-white callout"
                >
                                Brooklyn
                </h3>
</div>,
              <div
                className="flex flex-column"
>
                <h3
                                className="my2 text-white callout"
                >
                                Manhattan
                </h3>
</div>,
              <div
                className="flex flex-column"
>
                <h3
                                className="my2 text-white callout"
                >
                                Queens
                </h3>
</div>,
              <div
                className="flex flex-column"
>
                <h3
                                className="my2 text-white callout"
                >
                                New Jersey
                </h3>
</div>,
              <div
                className="flex flex-column"
>
                <h3
                                className="my2 text-white callout"
                >
                                Florida
                </h3>
</div>,
            ],
            "className": "flex flex-column flex-wrap",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "Brooklyn",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="my2 text-white callout"
>
                    Brooklyn
</h3>,
                  Array [],
                ],
                "className": "flex flex-column",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Brooklyn",
                    "className": "my2 text-white callout",
                  },
                  "ref": null,
                  "rendered": "Brooklyn",
                  "type": "h3",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "Manhattan",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="my2 text-white callout"
>
                    Manhattan
</h3>,
                  Array [],
                ],
                "className": "flex flex-column",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Manhattan",
                    "className": "my2 text-white callout",
                  },
                  "ref": null,
                  "rendered": "Manhattan",
                  "type": "h3",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "Queens",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="my2 text-white callout"
>
                    Queens
</h3>,
                  Array [],
                ],
                "className": "flex flex-column",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Queens",
                    "className": "my2 text-white callout",
                  },
                  "ref": null,
                  "rendered": "Queens",
                  "type": "h3",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "New Jersey",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="my2 text-white callout"
>
                    New Jersey
</h3>,
                  Array [],
                ],
                "className": "flex flex-column",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "New Jersey",
                    "className": "my2 text-white callout",
                  },
                  "ref": null,
                  "rendered": "New Jersey",
                  "type": "h3",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "Florida",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="my2 text-white callout"
>
                    Florida
</h3>,
                  Array [],
                ],
                "className": "flex flex-column",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Florida",
                    "className": "my2 text-white callout",
                  },
                  "ref": null,
                  "rendered": "Florida",
                  "type": "h3",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`renders without data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FooterLocations
    locations={Array []}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="mb2 text-white block-headline"
>
          Locations
</h2>,
        <div
          className="flex flex-column flex-wrap"
>
          <div
                    className="flex flex-column"
          >
                    <h3
                              className="my2 text-white callout"
                    >
                              Brooklyn
                    </h3>
          </div>
          <div
                    className="flex flex-column"
          >
                    <h3
                              className="my2 text-white callout"
                    >
                              Manhattan
                    </h3>
          </div>
          <div
                    className="flex flex-column"
          >
                    <h3
                              className="my2 text-white callout"
                    >
                              Queens
                    </h3>
          </div>
          <div
                    className="flex flex-column"
          >
                    <h3
                              className="my2 text-white callout"
                    >
                              New Jersey
                    </h3>
          </div>
          <div
                    className="flex flex-column"
          >
                    <h3
                              className="my2 text-white callout"
                    >
                              Florida
                    </h3>
          </div>
</div>,
      ],
      "className": "flex flex-column m4",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Locations",
          "className": "mb2 text-white block-headline",
        },
        "ref": null,
        "rendered": "Locations",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="flex flex-column"
>
              <h3
                            className="my2 text-white callout"
              >
                            Brooklyn
              </h3>
</div>,
            <div
              className="flex flex-column"
>
              <h3
                            className="my2 text-white callout"
              >
                            Manhattan
              </h3>
</div>,
            <div
              className="flex flex-column"
>
              <h3
                            className="my2 text-white callout"
              >
                            Queens
              </h3>
</div>,
            <div
              className="flex flex-column"
>
              <h3
                            className="my2 text-white callout"
              >
                            New Jersey
              </h3>
</div>,
            <div
              className="flex flex-column"
>
              <h3
                            className="my2 text-white callout"
              >
                            Florida
              </h3>
</div>,
          ],
          "className": "flex flex-column flex-wrap",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "Brooklyn",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="my2 text-white callout"
>
                  Brooklyn
</h3>,
                Array [],
              ],
              "className": "flex flex-column",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Brooklyn",
                  "className": "my2 text-white callout",
                },
                "ref": null,
                "rendered": "Brooklyn",
                "type": "h3",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "Manhattan",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="my2 text-white callout"
>
                  Manhattan
</h3>,
                Array [],
              ],
              "className": "flex flex-column",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Manhattan",
                  "className": "my2 text-white callout",
                },
                "ref": null,
                "rendered": "Manhattan",
                "type": "h3",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "Queens",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="my2 text-white callout"
>
                  Queens
</h3>,
                Array [],
              ],
              "className": "flex flex-column",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Queens",
                  "className": "my2 text-white callout",
                },
                "ref": null,
                "rendered": "Queens",
                "type": "h3",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "New Jersey",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="my2 text-white callout"
>
                  New Jersey
</h3>,
                Array [],
              ],
              "className": "flex flex-column",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "New Jersey",
                  "className": "my2 text-white callout",
                },
                "ref": null,
                "rendered": "New Jersey",
                "type": "h3",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": "Florida",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="my2 text-white callout"
>
                  Florida
</h3>,
                Array [],
              ],
              "className": "flex flex-column",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Florida",
                  "className": "my2 text-white callout",
                },
                "ref": null,
                "rendered": "Florida",
                "type": "h3",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="mb2 text-white block-headline"
>
            Locations
</h2>,
          <div
            className="flex flex-column flex-wrap"
>
            <div
                        className="flex flex-column"
            >
                        <h3
                                    className="my2 text-white callout"
                        >
                                    Brooklyn
                        </h3>
            </div>
            <div
                        className="flex flex-column"
            >
                        <h3
                                    className="my2 text-white callout"
                        >
                                    Manhattan
                        </h3>
            </div>
            <div
                        className="flex flex-column"
            >
                        <h3
                                    className="my2 text-white callout"
                        >
                                    Queens
                        </h3>
            </div>
            <div
                        className="flex flex-column"
            >
                        <h3
                                    className="my2 text-white callout"
                        >
                                    New Jersey
                        </h3>
            </div>
            <div
                        className="flex flex-column"
            >
                        <h3
                                    className="my2 text-white callout"
                        >
                                    Florida
                        </h3>
            </div>
</div>,
        ],
        "className": "flex flex-column m4",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Locations",
            "className": "mb2 text-white block-headline",
          },
          "ref": null,
          "rendered": "Locations",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="flex flex-column"
>
                <h3
                                className="my2 text-white callout"
                >
                                Brooklyn
                </h3>
</div>,
              <div
                className="flex flex-column"
>
                <h3
                                className="my2 text-white callout"
                >
                                Manhattan
                </h3>
</div>,
              <div
                className="flex flex-column"
>
                <h3
                                className="my2 text-white callout"
                >
                                Queens
                </h3>
</div>,
              <div
                className="flex flex-column"
>
                <h3
                                className="my2 text-white callout"
                >
                                New Jersey
                </h3>
</div>,
              <div
                className="flex flex-column"
>
                <h3
                                className="my2 text-white callout"
                >
                                Florida
                </h3>
</div>,
            ],
            "className": "flex flex-column flex-wrap",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "Brooklyn",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="my2 text-white callout"
>
                    Brooklyn
</h3>,
                  Array [],
                ],
                "className": "flex flex-column",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Brooklyn",
                    "className": "my2 text-white callout",
                  },
                  "ref": null,
                  "rendered": "Brooklyn",
                  "type": "h3",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "Manhattan",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="my2 text-white callout"
>
                    Manhattan
</h3>,
                  Array [],
                ],
                "className": "flex flex-column",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Manhattan",
                    "className": "my2 text-white callout",
                  },
                  "ref": null,
                  "rendered": "Manhattan",
                  "type": "h3",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "Queens",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="my2 text-white callout"
>
                    Queens
</h3>,
                  Array [],
                ],
                "className": "flex flex-column",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Queens",
                    "className": "my2 text-white callout",
                  },
                  "ref": null,
                  "rendered": "Queens",
                  "type": "h3",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "New Jersey",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="my2 text-white callout"
>
                    New Jersey
</h3>,
                  Array [],
                ],
                "className": "flex flex-column",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "New Jersey",
                    "className": "my2 text-white callout",
                  },
                  "ref": null,
                  "rendered": "New Jersey",
                  "type": "h3",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": "Florida",
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="my2 text-white callout"
>
                    Florida
</h3>,
                  Array [],
                ],
                "className": "flex flex-column",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Florida",
                    "className": "my2 text-white callout",
                  },
                  "ref": null,
                  "rendered": "Florida",
                  "type": "h3",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
